models/survey.py

```diff
@@ -17,36 +17,8 @@ def __init__(self, id_, description_):
         self.description = description_
 
     def __repr__(self):
-        return "<Question: {}>".format(self.name)
+        return "<Survey: {}>".format(self.id)
 
 class surveyForm(Form):
```

Comment:
Does this deserve to be included in models? Maybe each form class should have its own file in `/forms`.

Reply:
I dont know, can be changed any way :)